app/portainer/components/sidebar/SidebarMenuItem/SidebarMenuItem.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Story } from '@storybook/react';
+
+import { SidebarMenuItem } from './SidebarMenuItem';
+
+const meta: Meta = {
+  title: 'Components/SidebarMenuItem',
+  component: SidebarMenuItem,
+};
+export default meta;
+
+interface StoryProps {
+  iconClass: string;
+  className: string;
+  itemName: string;
+  LinkName: string;
+}
+
+function Template({ iconClass, className, itemName, LinkName }: StoryProps) {
+  return (
+    <ul className="sidebar">
+      <div className="sidebar-list">
+        <SidebarMenuItem
+          path="example.path"
+          pathParams={{ endpointId: 1 }}
+          iconClass={iconClass}
+          className={className}
+          itemName={itemName}
+        >
+          {LinkName}
+        </SidebarMenuItem>
+      </div>
+    </ul>
+  );
+}
+
+export const Primary: Story<StoryProps> = Template.bind({});
+Primary.args = {
+  iconClass: 'fa-tachometer-alt fa-fw',
+  className: 'exampleItemClass',
+  itemName: 'ExampleItem',
+  LinkName: 'Example Link Name',
+};
+
+export function WithoutLinkName() {
+  return (
+    <SidebarMenuItem
+      path="example.path"
+      pathParams={{ endpointId: 1 }}
+      iconClass="fa-tachometer-alt fa-fw"
+      className="exampleItemClass"
+      itemName="ExampleItem"
+    />
+  );
+}
+
+export function WithoutIconOrLinkName() {
+  return (
+    <SidebarMenuItem
+      path="example.path"
+      pathParams={{ endpointId: 1 }}
+      className="exampleItemClass"
+      itemName="ExampleItem"
+    />
+  );
+}
```

Comment:
```suggestion
export const Primary: Story<StoryProps> = Template.bind({});
Primary.args = {
  iconClass: '',
  linkName: 'example',
};
```

Reply:
Removed the irrelevant story and updated the story with the icon, as there is many places we use sidebar menu items without an icon

![image](https://user-images.githubusercontent.com/12197793/156248785-739f8092-459c-4f9c-a5d6-44e085599e7d.png)